.cirrus.yml

```diff
@@ -44,10 +47,20 @@ rhel_8_task:
     matrix:
       - SETUP: setup.py develop --user
       - SETUP: -m pip install .
+  # Latest pip available avocado-71.0 contains a bug
+  # Latest pip available 69.1 contains a bug
+  allow_failures: $AVOCADO_SRC == '' || $AVOCADO_SRC == 'avocado-framework<70.0'
+  skip_notifications: $AVOCADO_SRC == '' || $AVOCADO_SRC == 'avocado-framework<70.0'
   setup_script:
     *setup_scr
-  plugin_script:
-    - $PYTHON -m avocado plugins 2>/dev/null | grep vt-bootstrap
+  mock_deps_script:
+    - for PROG in tcpdump qemu-kvm qemu-img qemu-io; do touch /usr/local/bin/$PROG; chmod +x /usr/local/bin/$PROG; done
```

Comment:
If we choose to fake the programs, we can do that just once in the container images.

Reply:
good point